editor/src/components/canvas/canvas-strategies/absolute-reparent-strategy.spec.browser2.tsx

```diff
@@ -326,9 +326,7 @@ describe('Absolute Reparent Strategy', () => {
 
     await renderResult.getDispatchFollowUpActionsFinished()
 
-    expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(
-      createCodeForProject(-960, -950),
-    )
+    expect(getPrintedUiJsCode(renderResult.getEditorState())).toEqual(createCodeForProject(40, 50))
```

Comment:
this line looks sus to me

Reply:
This is because of the `ifAllowedToReparent` that was previously missing from the absolute reparent strategy (but present in all flex reparenting strategies). Previously the absolute reparent would actually be applied, but wouldn't _actually_ reparent, meaning it was acting as an absolute move instead (whilst the picker would still claim it was performing a reparent). Now this will instead show the "not allowed" cursor, and will leave the element as it is (unless you explicitly switch to the "Move" strategy)